tests/appsec/test_blocking_addresses.py

```diff
@@ -600,6 +468,22 @@ def test_non_blocking(self):
         for code, response in self.rm_req_nonblock.items():
             assert response.status_code == code, response.request.url
 
+    def setup_not_found(self):
+        self.rnf_req = weblog.get(path="/finger_print")
+
+    @missing_feature(
+        context.library == "java" and context.weblog_variant not in ("akka-http", "play"),
+        reason="Happens on a subsequent WAF run",
+    )
+    @missing_feature(context.library == "ruby", reason="Not working")
+    @missing_feature(context.library < "nodejs@5.17.0", reason="Not supported yet")
```

Comment:
this whole `Test_Blocking_response_status` class is already set as `5.17.0` in the node manifest
```suggestion
```